src/gt4py/next/embedded/nd_array_field.py

```diff
@@ -159,6 +160,10 @@ def from_array(
     def remap(
         self: NdArrayField, connectivity: common.ConnectivityField | fbuiltins.FieldOffset
     ) -> NdArrayField:
+        # Current implementation relies on SKIP_VALUE == -1:
+        # if we assume the indexed array has at least one element, we wrap around without out of bounds
+        assert common.SKIP_VALUE == -1
```

Comment:
shouldn't this assert the skip_value of the connectivity?

Reply:
yes, fixed.